### ISSUE TITLE
Traceback spam from ElementArrayStim

### ISSUE THREAD

**Issue by TEParsons** [psychopy/psychopy#3327]
When using a Slider, if an error occurs elsewhere in the experiment, it causes something to go wrong with the __del__ method of ElementArrayStim, meaning that after the informative traceback for the actual error:
```
Traceback (most recent call last):
  File "F:\OneDrive\OneDrive - Open Science Tools\Throwaway\sliderBreak_lastrun.py", line 136, in <module>
    1/0
ZeroDivisionError: division by zero
```
there is a load of extra:
```
Exception ignored in: <bound method ElementArrayStim.__del__ of <psychopy.visual.elementarray.ElementArrayStim object at 0x000001D64F46E4E0>>
Traceback (most recent call last):
  File "C:\Program Files\PsychoPy3\lib\site-packages\psychopy\visual\elementarray.py", line 735, in __del__
  File "C:\Program Files\PsychoPy3\lib\site-packages\psychopy\visual\basevisual.py", line 1020, in clearTextures
  File "C:\Program Files\PsychoPy3\lib\site-packages\pyglet\gl\lib.py", line 97, in errcheck
ImportError: sys.meta_path is None, Python is likely shutting down
```

Attached minimal example:
[sliderBreak.zip](https://github.com/psychopy/psychopy/files/5607862/sliderBreak.zip)

**Comment by peircej** [comment]
I think this is fixed already. Just checking

**Comment by peircej** [comment]
Hmmm, not quite the same bug, but caused by the same general issue as fixed in ebfe7c9273b951aca7910f7ac5b9184a6aed96a7 

Almost certainly it means that something in pyglet has been garbage-collected before the stimulus is garbage-collected and that made it hard to clear away (the stimulus wants pyglet to clear graphics memory but pyglet is already gone)
